docs/src/solvers/sde_solve.md

```diff
@@ -64,6 +64,13 @@ Orders are given in terms of strong order.
   all forms of noise, including non-diagonal, scalar, and colored noise.†
 - `EulerHeun` - The Euler-Heun method. Strong Order 0.5 in the Stratonovich sense.
   Can handle all forms of noise, including non-diagonal, scalar, and colored noise.†
+- `PCEuler` - The predictor corrector euler method. Strong Order 0.5 in the Ito
+  sense. Requires the bbprime function, which is defined as
+  ```math
```

Comment:
Move this above to a section, like commutative noise.

Reply:
Made a new "Analytic Methods" section. Presumably, things like the Mil with analytic derivatives will fall under this method as well. What do you think?